app/helpers/provider/admin/dashboards_helper.rb

```diff
@@ -1,5 +1,7 @@
 module Provider::Admin::DashboardsHelper
 
+  include ApplicationHelper
```

Comment:
Why do we need ApplicationHelper?

Reply:
It's making use of `ApplicationHelper.css_class`

Reply:
will be available anyway

Reply:
OK probably for sanity in the tests, it is best to put the dependencies here.
Not really your fault.